content/en/docs/concepts/services-networking/service.md

```diff
@@ -528,7 +528,7 @@ and `.spec.clusterIP:spec.ports[*].port`. (If the `--nodeport-addresses` flag in
 ### Type LoadBalancer {#loadbalancer}
 
 On cloud providers which support external load balancers, setting the `type`
-field to `LoadBalancer` provisions a load balancer for your Service.
+field to `LoadBalancer` provisions a load balancer for your Service. 
```

Comment:
Just for future reference, it's best to avoid mixing whitespace/style-only changes with content changes. Detecting that a particular change is whitespace/style-only adds to reviewer overhead. :)

Reply:
Urgh. Missed that whitespace addition. Thanks @jaypipes